aspnetcore/grpc/supported-platforms.md

```diff
@@ -0,0 +1,62 @@
+---
+title: gRPC for .NET supported platforms
+author: jamesnk
+description: Learn about the supported platforms for gRPC on .NET.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.date: 01/22/2021
+no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: grpc/supported-platforms
+---
+# gRPC for .NET supported platforms
+
+By [James Newton-King](https://twitter.com/jamesnk)
+
+## Device requirements
+
+gRPC for .NET supports any device that .NET Core supports.
+
+> [!div class="checklist"]
+>
+> * Windows
+> * Linux
+> * macOS&dagger;
+
+&dagger;ASP.NET Core websites on macOS doesn't support HTTPS. gRPC clients on macOS can still use HTTPS when calling remote services.
+
+## ASP.NET Core server requirements
+
+gRPC services can be hosted on all built-in ASP.NET Core servers.
+
+> [!div class="checklist"]
+>
+> * Kestrel
+> * TestServer
+> * IIS&dagger;
+> * HttpSys&dagger;
+
+&dagger;IIS and HttpSys require .NET 5 and Windows XXXX or later
```

Comment:
don't forget to update XXXX

Reply:
Windows 10 Build 20241 or newer

@jkotalik Does IIS work with .NET 5

Reply:
Yes

Reply:
👍